front/package.json

```diff
@@ -69,6 +69,7 @@
     "maplibre-gl": "^4.0.0",
     "openapi-typescript-codegen": "^0.29.0",
     "party-js": "^2.2.0",
+    "pdf-parse": "^1.1.1",
```

Comment:
This should not be a runtime dep, since it's used only in tests. It should be a dev dep.